projects/core/src/product/store/effects/product.effect.ts

```diff
@@ -71,6 +72,12 @@ export class ProductEffects {
     );
   }
 
+  @Effect()
+  clearProductPrice$: Observable<ProductActions.ClearProductPrice> = this.actions$.pipe(
+    ofType(AuthActions.LOGOUT, AuthActions.LOGIN),
+    map(() => new ProductActions.ClearProductPrice())
+  );
+
```

Comment:
Instead of a new effect/action/scope, could we simply have use a reloading trigger in the existing scopes that carry the price?  https://sap.github.io/spartacus-docs/loading-scopes/#reloading-triggers
I think there are login/logout events we could use for that.

Reply:
This is good! But we will reloadOn `price`.